model.py

```diff
@@ -356,7 +336,10 @@ def get_word_chars(sentence, i2w, c2i):
     logging.info("Number training instances: {}".format(len(training_instances)))
     logging.info("Number dev instances: {}".format(len(dev_instances)))
 
-    for epoch in xrange(int(options.num_epochs)):
+    best_dev_pos = 0.0
+    old_best_name = None
+
+    for epoch in range(int(options.num_epochs)):
```

Comment:
is this int casting still required? maybe make the `num_epochs` param into an `int` value?